.travis.yml

```diff
@@ -24,3 +24,5 @@ after_success:
   - bin/createcoverage
   - pip install coverage==3.7.1 coveralls
   - coveralls
+  - docker.io build --tag plone.restapi:$(git rev-parse HEAD) .
+#  - docker.io push kitconcept/plone.restapi
```

Comment:
Reminder: Do not forget to change that to plone, later on, otherwise you will push it to kitconcept on dockr hub and not to plone :)